components/script/dom/htmliframeelement.rs

```diff
@@ -228,6 +228,8 @@ impl HTMLIFrameElement {
     /// <https://html.spec.whatwg.org/multipage/#process-the-iframe-attributes>
     fn process_the_iframe_attributes(&self, mode: ProcessingMode) {
         // TODO: srcdoc
+        // Still remaining because we are understanding the specification:
+        // https://html.spec.whatwg.org/multipage/#process-the-iframe-attributes
```

Comment:
You will want to:
* get the srcdoc attribute value
* if the srcdoc attribute value exists, create a LoadData instance that contains the attribute's value and the url about:srcdoc
* call self.navigate_or_reload_child_browsing_context with the new loaddata
* return without executing the rest of the method

Reply:
Implemented the requested changes. It'd be really helpful if you could review it as tests are still failing.